generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-dialog.component.html

```diff
@@ -36,8 +36,8 @@ <h4 class="modal-title" id="my<%= entityClass %>Label" data-translate="<%= keyPr
             } else if ((fieldType == 'byte[]' || fieldType == 'ByteBuffer') && fieldTypeBlobContent != 'text') {
                 fieldInputType = 'hidden';
             } _%>
-        <div class="form-group"<% if ((fieldType == 'byte[]' || fieldType === 'ByteBuffer') && fieldTypeBlobContent != 'text') { %> ngf-drop ngf-change="vm.set<%=fieldNameCapitalized %>($file, vm.<%= entityInstance %>)"<% if (fieldTypeBlobContent == 'image') { %> ngf-pattern="'image/*'"<% } %><% } %>>
-            <label class="control-label" data-translate="<%= translationKey %>" for="field_<%= fieldName %>"><%= fieldNameHumanized %></label>
+        <div class=<% if (fieldType === 'Boolean') { %>"form-check"<% } else {%>"form-group"<% } %><% if ((fieldType == 'byte[]' || fieldType === 'ByteBuffer') && fieldTypeBlobContent != 'text') { %> ngf-drop ngf-change="vm.set<%=fieldNameCapitalized %>($file, vm.<%= entityInstance %>)"<% if (fieldTypeBlobContent == 'image') { %> ngf-pattern="'image/*'"<% } %><% } %>>
```

Comment:
may be we can move this ugly spagetti to an ejs block and just assign the result variable to the class here?